src/components/App.css

```diff
@@ -0,0 +1,22 @@
+
+.app {
+  background-color: #0E0E0E;
+  color: white;
+  font-family: 'Montserrat', sans-serif;
+  letter-spacing: 2px;
+} 
+
+h1 {
+  font-family: 'Edo-sz';
+}
+
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.landing-page {
```

Comment:
I can't find where this class is applied.

Reply:
That was the previous name of the class, it has now been changed to the correct name.